src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas3/ReportService.kt

```diff
@@ -18,13 +19,19 @@ class ReportService(
   private val offenderService: OffenderService,
   private val userService: UserService,
   private val transitionalAccommodationReferralReportRowRepository: TransitionalAccommodationReferralReportRepository,
+  @Value("\${cas3-report.end-date-override:0}") private val endDateOverride: Int,
```

Comment:
i'd much rather we used null to represent absence of value instead of 0, if at all possible

Reply:
I defaulted to zero instead of null to avoid if anyone sets the property value to 0.

Reply:
I think that's addressing two different concerns though?

1. Shouldn't allow someone to set it to 0
2. Default behaviour when not set

IMO opinion it would be clearer if default was null but as this is presumably temp code can leave as is